src/libsync/owncloudpropagator.h

```diff
@@ -188,18 +188,18 @@ class OWNCLOUDSYNC_EXPORT PropagateDirectory : public PropagatorJob {
     QScopedPointer<PropagateItemJob>_firstJob;
 
     // all the sub files or sub directories.
-    QVector<PropagatorJob *> _subJobs;
+    QList<PropagatorJob *> _subJobs;
```

Comment:
You can delete from the front with QVector and we might want to go for memory over speed here. But I don't have confidence of the best choice here. QVector should be fine.